fastlane/lib/fastlane/actions/ssh.rb

```diff
@@ -6,7 +6,7 @@ module SharedValues
     end
 
     class SshAction < Action
-      def self.ssh_exec!(ssh, command)
+      def self.ssh_exec!(ssh, command, log)
```

Comment:
Happy to do that, but I'm not sure I understand – the only call site for `ssh_exec` that I can find is in `ssh.rb`, which has been changed to use `params[:log]`. Who would be calling `ssh_exec`, and how? 

This might be another Ruby thing I don't yet get, so humor me. :)